app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListFragment.kt

```diff
@@ -293,8 +293,8 @@ class StarredListFragment :
     }
 
     private fun deleteSession(session: Session) {
```

Comment:
Side note: This doesn't appear to actually delete a session. So maybe the method should be renamed. Probably best done in a separate PR.

Reply:
:x: I leave the naming in the fragment (UI). In the scope of the screen deleting a session makes sense.

Reply:
You're removing a session from the list in that screen. That's not the same thing as deleting a session.

Reply:
:heavy_check_mark: Function inlined.